README.md

```diff
@@ -445,7 +464,9 @@ This figure presents an example of categorizing a known lab-cultured
 sample, along with the corresponding minimal metadata. The
 bidirectionality of the categorization framework is highlighted, as it
 enables the linkage between known, commercially available cultures and
-their original sample environments. <br><br>
+their original sample environments.
+
+
 
 ## <span id="license-ontology">Licenses and ontologies section</span>
```

Comment:
```suggestion
#<span id="license-ontology">Licenses and ontologies section</span>
```
1 hashtag as it was done for the other spanning sections

Reply:
Thanks, but IMO, the span HTML tags should be removed as well and the anchor links (for navigation) should be written in proper Markdown syntax, example below.

```markdown
[Licenses and ontologies section](#license-ontology)
```